### DISCUSSION TITLE
Inclusive Naming Initiative

### DISCUSSION THREAD

**Discussion by cognitivegears** [get-woke/woke#127] [Ideas] [answered]
Not exactly a software idea, but I know you said that you were doing some work on the website. Once that is complete, you may want to consider seeing if the Inclusive Naming Initiative (inclusivenaming.org) would add `woke` as a resource and provide a link to the project. Looks like they have a growing list of participating organizations, it may be a good way to get the word out about the tool.

**Comment by caitlinelfring** [comment, marked as answer]
https://github.com/inclusivenaming/org/pull/73

I tried and based on their response, I think I will be waiting a while until they mature their policies some more and are on the same page internally. Thanks for the suggestion, I've said all I intend to say on this matter.

**Reply by cognitivegears** [reply]
Wow, that's a shame and sorry you had a not so great experience!
